src/Umbraco.Infrastructure/Cache/DistributedCacheBinder_Handlers.cs

```diff
@@ -66,11 +66,6 @@ public void BindEvents(bool supportUnbinding = false)
             Bind(() => UserService.UserGroupPermissionsAssigned += UserService_UserGroupPermissionsAssigned,
                 () => UserService.UserGroupPermissionsAssigned -= UserService_UserGroupPermissionsAssigned);
 
-            // bind to data type events
-            Bind(() => DataTypeService.Deleted += DataTypeService_Deleted,
-                () => DataTypeService.Deleted -= DataTypeService_Deleted);
-            Bind(() => DataTypeService.Saved += DataTypeService_Saved,
-                () => DataTypeService.Saved -= DataTypeService_Saved);
```

Comment:
We need to hook up to these events again using the new pattern, like we did with LanguageSaved/Deleted etc.